hub/policy/src/test/java/uk/gov/ida/hub/policy/PolicyApplicationTest.java

```diff
@@ -25,9 +25,9 @@ public void setUp() throws Exception {
     }
 
     @Test
-    public void shouldReturnSendToRecordingSystemAsFalseByDefault() throws Exception {
+    public void shouldReturnEventEmitterConfigurationAsNullByDefault() throws Exception {
```

Comment:
ahh - the answer to me previous comment!
New question then - why would we ever want this to be null? Would we not want to fail fast if we couldn't create this?

Reply:
We do not want to fail fast. Instead, it will use stub queue by default when queue name is not provided i.e. event emitter will know that EventEmitterConfiguration is Optional.empty() and will use stub queue instead.

Reply:
ah right. I see. I was kind of imagining still having a configuration, but the configuration explicitly saying it didn't have a queueName.

I generally dislike configuring things by the absence of stuff, but I think you are right that this probably works just fine by now.